src/pages/my_page/user_profile/UserIdentity.tsx

```diff
@@ -0,0 +1,98 @@
+import VisibilityIcon from '@mui/icons-material/Visibility';
+import { getMaskedNationalIdentityNumber } from '../../../utils/user-helpers';
+import { UserRoles } from './UserRoles';
+
+interface UserIdentitiesProps {
+  user: User;
+}
+
+export const UserIdentity = ({ user }: UserIdentitiesProps) => {
+  const { t } = useTranslation();
+  const fullName = `${user.givenName} ${user.familyName}`;
+  const userCristinId = user.cristinId?.split('/').at(-1);
+  const nationalId = user.nationalIdNumber;
+  const [showFullNin, setShowFullNin] = useState(false);
+
+  return (
+    <BackgroundDiv sx={{ bgcolor: 'secondary.dark' }}>
+      <Typography variant="h2">{t('my_page.my_profile.identity.identity')}</Typography>
+      <Box
+        sx={{
+          display: 'grid',
+          gridTemplateColumns: {
+            xs: '1fr',
+            md: '4fr 3fr',
+          },
+          columnGap: '2rem',
+          rowGap: { xs: '1rem' },
+          mt: '1rem',
+        }}>
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', maxWidth: '30rem' }}>
+          <Typography>{t('my_page.my_profile.identity.not_shown_publicly')}</Typography>
+          <TextField
+            label={t('my_page.my_profile.identity.full_name')}
+            value={fullName}
+            size="small"
+            variant="filled"
+            disabled
+          />
+          <Box
+            sx={{
+              display: 'flex',
+              flexDirection: { xs: 'column', md: 'row' },
+              gap: { xs: '1rem', md: '5rem' },
+            }}>
+            <TextField
+              size="small"
+              label={t('my_page.my_profile.identity.national_identity_number')}
+              variant="filled"
+              disabled
+              value={showFullNin ? nationalId : getMaskedNationalIdentityNumber(nationalId)}
+              InputProps={{
+                endAdornment: (
+                  <Tooltip
+                    title={
+                      showFullNin
+                        ? t('my_page.my_profile.identity.hide_full_nin')
+                        : t('my_page.my_profile.identity.show_full_nin')
+                    }>
+                    <IconButton onClick={() => setShowFullNin(!showFullNin)}>
+                      {showFullNin ? <VisibilityIcon /> : <VisibilityOffIcon />}
+                    </IconButton>
+                  </Tooltip>
+                ),
+              }}
+            />
+            <TextField
+              size="small"
+              label={'Cristin ID'}
+              value={userCristinId ? userCristinId : ''}
+              variant="filled"
+              disabled
+            />
+          </Box>
+        </Box>
+
+        <Box>
+          <Box
+            sx={{
+              backgroundColor: 'white',
+              display: 'flex',
+              flexDirection: 'column',
+              gap: '0.5rem',
+              pb: '0.5rem',
+              px: '0.5rem',
```

Comment:
Litt beskjeden padding her?
kanskje legge til en bitteliten borderRadius også?
```suggestion
              p: '1rem',
```